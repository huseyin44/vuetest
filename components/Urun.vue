<template>
 <v-app>
  <div>
      <v-row>
          <v-col class="aragenis"></v-col>
          <v-col class="aragenis"></v-col>
          <v-col class="aragenis"></v-col>
          <v-col class="sutunbir">
            <v-row class="imagebir">
             <img style="max-height:108px;max-width:108px;" src="https://cdn.saatvesaat.com.tr/media/catalog/product/a/x/ax2802_1.jpg">
            </v-row>
            <v-row class="arayuksek"></v-row>
            <v-row class="imagebir1">
             <img style="max-height:108px;max-width:108px;" src="https://cdn.saatvesaat.com.tr/media/catalog/product/a/x/ax2802_2.jpg">
            </v-row>
          </v-col>
          <v-col class="aragenis"></v-col>
          <v-col >
            <v-row class="imageiki">
             <img class="imageuc" src="https://cdn.saatvesaat.com.tr/media/catalog/product/a/x/ax2802_1.jpg">
            </v-row>
          </v-col>
          <v-col>
              <v-row>
                <v-col>
                 <nuxt-link to="/"  style="text-decoration: underline; color: inherit;white-space: nowrap;width:3%;">
                 Anasayfa
                 </nuxt-link>
                </v-col>
                <v-col style="width:0.1%">></v-col>
                <v-col>
                 <nuxt-link to="/MarkaClick"  style="text-decoration: underline; color: inherit;white-space: nowrap;width:3%;">
                 Saat
                 </nuxt-link>
                </v-col>
                <v-col style="width:0.1%">></v-col>
                <v-col>
                 <nuxt-link to="/MarkaClick"  style="text-decoration: underline; color: inherit;white-space: nowrap;width:3%;">
                 Erkek Saat Modelleri
                 </nuxt-link>
                </v-col>
                <v-col style="width:0.1%">></v-col>
                <v-col><div style="color:gray;white-space: nowrap;width:3%;">AX2802 Erkek Kol Saati</div></v-col>
              </v-row>
              <v-row class="satir11">
                  <v-col>
                      <v-row>
                        <img class="markaresim" src="https://cdn.saatvesaat.com.tr/media/amshopby/big_1735e392dea76f18.png">
                      </v-row>
                      <v-row class="arayuksek"></v-row>
                      <v-row>
                          <nuxt-link to="/MarkaClick"  style=" color: inherit;">
                          Tüm Armani Exchange Ürünleri
                          </nuxt-link>
                      </v-row>
                  </v-col>
                  <v-col class="aragenis2"></v-col>
                  <v-col class="aragenis2"></v-col>
                  <v-col class="aragenis2"></v-col>
                  <v-col class="aragenis2"></v-col>
                  <v-col class="aragenis2"></v-col>
                  <v-col>
                      <div style="color:black;font-size:110%;">Türkiye Yetkili Satıcısı</div>
                  </v-col>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:black;font-size:130%;">Armani Exchange</div>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:gray;font-size:150%;">AX2802 Erkek Kol Saati</div>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:gray;font-size:100%;">Çelik / Quartz Teknoloji / 42 mm Kasa Çapı</div>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:gray;font-size:120%;text-decoration:line-through;">337000 TL</div>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:red;font-size:170%;">151600 TL</div>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row class="arayuksek"></v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
              <v-col style="padding-top:9%"><div>Adet</div></v-col>
              <v-col style="padding-top:9%">
                <select style="width:35%">
                 <option selected="selected" value="1">1</option>
                 <option value="2">2</option>
                 <option value="3">3</option>
                 <option value="4">4</option>
                 <option value="5">5</option>
                 <option value="6">6</option>
                 <option value="7">7</option>
                 <option value="8">8</option>
                 <option value="9">9</option>
                 <option value="10">10</option>
                </select>
              </v-col>
              <v-col>
              <nuxt-link to="/Alisverisdevam">
              <button style="text-decoration:none"  class="satinal">
                SATIN AL
              </button>
              </nuxt-link>
              </v-col>
              <v-col class="aragenis2"></v-col><v-col class="aragenis2"></v-col><v-col class="aragenis2"></v-col><v-col class="aragenis2"></v-col>
              <v-col class="aragenis2"></v-col><v-col class="aragenis2"></v-col>
              </v-row>
              <v-row>
                  <div style="color:353535;font-family:Roboto, Arial, sans-serif;">Web'e özel Ürünlerin iade ve değişim işlemleri sadece online kanaldan sağlanmakta olup, 
                      mağazalarımızda değişim gerçekleştirilmemektedir.</div>
              </v-row>
              <v-row class="arayuksek"></v-row><v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:lightgray;font-family:Roboto, Arial, sans-serif;">Sorularınız ve destek için online chat veya 0212 367 44 74 </div>
              </v-row>
          </v-col>
      </v-row>
  </div>
 </v-app>
</template>

<script>

</script>

<style>
.sutunbir{
 max-width: 110px;
}
.arayuksek{
 height:4px;
}
.aragenis{
 width:4px;
 max-width:4px ;
}
.aragenis2{
 width:24px;
 max-width:24px ;
}
.imagebir{
 display: inline-block;
 border: 1px solid #e4e4e4;
}
.imagebir1{
 display: inline-block;
 border: 1px solid #e4e4e4;
}
.imageiki{
 border: 1px solid #e4e4e4;
 max-width: 532px;
 height: 500px;
}
.imageuc{
 max-height: 480px;
 max-width: 520px;
}
.markaresim{
 max-height: 48px;
 max-width: 216px;
}
.satir11{
border-top-style:solid ;
border-bottom:solid ;
border-right:none ;
border-left:none ;
border-color:gray;
border-width: 1px;
width: 580px;
}
.satinal{
min-height: 40px;
line-height: 30px;
font-size: 28px;
padding: 15px 30px;
background: black;
color: white;
min-width: 390px;
width: 100%;
white-space: normal;
font-weight: 500;
border: 4px solid #000000;
text-align: center;
}
.satinal:hover{
 color: black;
  background: white;
  border-style:solid;
  border-color:black;
}
</style>