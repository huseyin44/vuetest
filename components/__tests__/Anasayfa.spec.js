import { shallowMount,RouterLinkStub, mount } from "@vue/test-utils";
import Urun from "../Anasayfa";
import Vue from 'vue'
import Vuetify from 'vuetify'
import assert from 'assert'
Vue.use(Vuetify)
const factory = () => {
  return shallowMount(Urun, {});
};
describe('Urun', () => {
    let vuetify
    beforeEach(() => {
      vuetify = new Vuetify()
    })  
    test('NuxtLink Ozelligi', () => {
      const wrapper = shallowMount(Urun, {
        stubs: {
          NuxtLink: RouterLinkStub,
        },
      })
      const links = wrapper.findAll(RouterLinkStub)
      assert(links.at(0).props().to === '/')
      assert(links.at(1).text() === 'Armani Exchange')
      assert(links.at(1).props().to === '/MarkaClick')
      assert(links.at(2).text() === 'Giris')
      assert(links.at(2).props().to === '/Giris')
    })
    it('Bir Vue kopmonenti', () => {
      const wrapper = factory();
      expect(wrapper.isVueInstance()).toBeTruthy();
  })
    it('Snapshot Ozelligi', () => {
      const wrapper = factory();
      expect(wrapper.html()).toMatchSnapshot();
  })
    test('triggers a mouseover', async () => {
      const wrapper = shallowMount(Urun)
      await wrapper.trigger('mouseover')
    })
   test('triggers a mouseleave', async () => {
      const wrapper = shallowMount(Urun)
      await wrapper.trigger('mouseleave')
    })
  
 }) 
