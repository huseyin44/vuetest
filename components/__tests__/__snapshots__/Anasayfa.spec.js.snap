// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Urun Snapshot Ozelligi 1`] = `
<div id="anasayfa">
  <div class="test">%100 Orijinal • Ücretsiz Kargo • Vade Farksız 9 Taksit </div>
  <v-container-stub tag="div" fluid="true" class="birincisatir">
    <v-row-stub tag="div">
      <v-col-stub tag="div">
        <nuxt-link to="/">
          <v-img-stub options="[object Object]" position="center center" src="https://cdn.saatvesaat.com.tr/skin/frontend/base/default/images/new-logo.png" transition="fade-transition" class="saat"></v-img-stub>
        </nuxt-link>
      </v-col-stub>
      <v-col-stub tag="div" class="buton1">BAYAN SAAT</v-col-stub>
      <v-col-stub tag="div" class="buton1">ERKEK SAAT</v-col-stub>
      <v-col-stub tag="div" class="buton1">ÇOCUK SAAT</v-col-stub>
      <v-col-stub tag="div" class="buton10">
        <v-row-stub tag="div" justify="center">
          <v-menu-stub opendelay="0" closedelay="0" contentclass="" openonhover="true" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" openonclick="true" closeonclick="true" closeoncontentclick="true" maxheight="auto" offsety="true" origin="top left" transition="v-menu-transition">
            <v-list-stub tag="div" class="itemlistelemecolumn">
              <div class="red3">
                88 Rue Du Rhone
              </div>
              <nuxt-link to="/MarkaClick" style="text-decoration: none;">
                <div class="red3">Armani Exchange </div>
              </nuxt-link>
              <div class="red3">Bomberg </div>
              <div class="red3">Boss Watches </div>
              <div class="red3">DKNY </div>
              <div>Edox </div>
              <div>Empori Armani </div>
              <div>Escape </div>
              <div>Esprit </div>
              <div>FENDI </div>
              <div>Fossil</div>
              <div>Gc </div>
              <div>Glam Rock </div>
              <div>Guess </div>
              <div>Lacoste </div>
              <div>Lorus </div>
              <div>Oris </div>
              <div>Parigi </div>
              <div>Seiko </div>
              <div>Skagen </div>
              <div>Skechers </div>
              <div>Tag HEUER </div>
              <div>Thom Olson </div>
              <div>Young Sterna </div>
              <div>Xonix </div>
            </v-list-stub>
          </v-menu-stub>
        </v-row-stub>
      </v-col-stub>
      <v-col-stub tag="div" class="buton2">TAKI &amp; AKSESUAR</v-col-stub>
      <v-col-stub tag="div" class="buton4">FIRSAT</v-col-stub>
      <v-col-stub tag="div" class="buton3">ONLİNE ÖZEL</v-col-stub>
      <v-col-stub tag="div" class="aramaa">
        <div><input type="search" placeholder="Kelime / Kod" autocomplete="off" class="arama2"> <input type="image" src="https://www.flaticon.com/svg/static/icons/svg/622/622669.svg" class="arares"></div>
      </v-col-stub>
      <v-col-stub tag="div" class="buton7">
        <v-row-stub tag="div" justify="center" class="test2">
          <v-menu-stub opendelay="0" closedelay="0" contentclass="" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" openonclick="true" closeonclick="true" closeoncontentclick="true" maxheight="auto" offsety="true" origin="top left" transition="v-menu-transition">
            <v-list-stub tag="div">
              <div class="red9">Favori Listem</div>
              <div class="red9">Sepetim</div>
              <div class="red9">Alışverişi Tamamla</div>
              <div class="red9">
                <nuxt-link to="/Giris" style="text-decoration: none;">Giris</nuxt-link>
              </div>
              <div class="red9">Üye Ol</div>
            </v-list-stub>
          </v-menu-stub>
        </v-row-stub>
      </v-col-stub>
      <v-col-stub tag="div" class="buton7">
        <v-row-stub tag="div" justify="center" class="test2">
          <v-menu-stub opendelay="0" closedelay="0" contentclass="" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" openonclick="true" closeonclick="true" closeoncontentclick="true" maxheight="auto" offsety="true" origin="top left" transition="v-menu-transition">
            <v-list-stub tag="div">
              <div>Sepet</div>
              <div>Sepetiniz Boş </div>
            </v-list-stub>
          </v-menu-stub>
        </v-row-stub>
      </v-col-stub>
    </v-row-stub>
  </v-container-stub>
</div>
`;
